text/0034-sorted-transaction-outputs.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: sorted_transaction_outputs
+- Start Date: 2022-04-04
+- MCIP PR: [mobilecoinfoundation/mcips#34](https://github.com/mobilecoinfoundation/mcips/pull/34)
+- Tracking Issue: original PR: https://github.com/mobilecoinfoundation/mobilecoin/pull/561
+
+# Summary
+[summary]: #summary
+
+For blocks with a version >= 3, all the transaction's outputs MUST be sorted by the public key value.
+
+# Motivation
+[motivation]: #motivation
+
+The main idea behind this feature is to prevent establishing the order of outputs that were added to the 
+transaction, thus making hard or impossible to identify the source.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The enclave transaction validation will now check if the TxOuts are sorted.
+It will return a transaction validation error if not.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Before block version 3, the transaction validator was checking on the following checks
```

Comment:
nit: this is fairly subjective, so ignore as you see fit.  
The only reason I mention it is that it makes the statement more concise, and *I* don't think it loses any meaning.

```suggestion
Before block version 3, the transaction validator was checking:
```